static/src/javascripts/projects/common/modules/ui/autoupdate.js

```diff
@@ -146,7 +145,7 @@ const updateBlocks = (opts, pollUpdates) => {
             .then(resp => {
                 count = resp.numNewBlocks;
 
-                if (pollUpdates && count > 0) {
```

Comment:
@alinaboghiu @abeddow91  removing this seems to have fixed the tab update count bug 🥳

Reply:
nice one!